src/style.css

```diff
@@ -1,3 +1,54 @@
 body {
   margin: 0;
 }
+
+header {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+}
+
+.logo {
+  margin-left: 50px;
+  padding-top: 10px;
+  margin-bottom: 10px;
```

Comment:
Be consistent with your vertical margins (Add only top or only bottom margin, but not both). You've added both `margin-left` and `margin-bottom` to the `.logo` class, which can lead to collapsing margins and inconsistency in spacing.